newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js

```diff
@@ -291,15 +306,16 @@ export const HomePage = React.memo<Props>(
             <TeamProvider>
               <div style={isMobile ? styles.mobileContainer : styles.container}>
                 <div style={styles.scrollableContainer}>
-                  {activeTab !== 'community' && !!announcements && (
-                    <AnnouncementsFeed canClose level="urgent" addMargins />
-                  )}
+                  {activeTab !== 'community' &&
+                    activeTab !== 'get-started' &&
+                    activeTab !== 'build' &&
+                    !!announcements && (
```

Comment:
I remember adding those checks to display the announcement only on certain pages.
Can you put this prop outside and name it `shouldDisplayAnnouncements` ? (clearer imo)